docs/reference/operator/overview.md

```diff
@@ -13,7 +13,7 @@ The diagram below showcases the different components that the K8sGPT Operator in
 
 ## Customising the Operator
 
-As with other Helm Charts, the K8sGPT Operator can be customised by modifying [the `values.yaml` file.](https://github.com/k8sgpt-ai/k8sgpt/blob/main/charts/k8sgpt/values.yaml)
+As with other Helm Charts, the K8sGPT Operator can be customised by modifying  the [ `values.yaml`](https://github.com/k8sgpt-ai/k8sgpt/blob/main/charts/k8sgpt/values.yaml) file.
```

Comment:
I feel like folks might miss the hyperlink part because of the quotes wrapper around the file name. I think something like this would be helpful

... the `values.yaml` [file here](https://github.com/k8sgpt-ai/k8sgpt/blob/main/charts/k8sgpt/values.yaml)

That might help us separate the hyperlink from the quotes section. What do you think?

Reply:
I don't think it will be more cleaner as the wrapping quotes for file name and hyperlink in other word but it is argumentative assumption and as they both having the same functionality as pointing to `values.yaml` file the current resolved also works.